presto-orc/src/main/java/com/facebook/presto/orc/metadata/ColumnStatistics.java

```diff
@@ -32,13 +37,51 @@ public ColumnStatistics(
             DateStatistics dateStatistics,
             DecimalStatistics decimalStatistics)
     {
+        this(numberOfValues, booleanStatistics, integerStatistics, doubleStatistics, stringStatistics, dateStatistics, decimalStatistics, true, null);
```

Comment:
Wouldn't it be better to add `ImmutableList.of()` (empty) instead of `null` in place of non-existing bloom filters?